tests/test_A0_initialization.py

```diff
@@ -93,14 +106,21 @@ def test_check_input_path_posix():
     assert folder == os.path.join(REPO_PATH, "inputs")
 
 
+OUTPUT_PATH = os.path.join(".", "tests", "other")
+
+
 class TestCommandLineInput:
 
     parser = initializing.create_parser()
 
-    def test_input_file(self):
-        parsed = self.parser.parse_args(["-i", "input_file"])
+    def test_input_json_file(self):
```

Comment:
not sure what this does?

Reply:
I think I adapted it from before, it can be deleted, I agree

Reply:
Ah, it is checking what happens if the command line args are read. okay.

Reply:
> > Thanks @Bachibouzouk, I have only two small comments:
> > ```
> > * Whenever test function names don't get too long I'd go for the standard "test_`name_of_function_you_want_to_test`_`what_do_you_test_in_this_function`().
> > ```
> 
> @SabineHaas - what is "too long" : 80, 100 or 120 characters?

80 I'd say..